ethpm/backends/registry.py

```diff
@@ -78,11 +82,23 @@ def parse_registry_uri(uri: str) -> RegistryURI:
     """
     Validate and return (authority, pkg name, version) from a valid registry URI
     """
+    from web3.auto.infura import w3
```

Comment:
It would be ideal if we could setup a test chain and install the ENS infrastructure on it easily so that we could test this stuff locally without hitting infura.  Maybe that's a project for the Web3.py codebase to support under something like `web3.tools` for doing one-step setup of all of the ENS contracts on a development chain.